test/smoke/dapi.js

```diff
@@ -1,10 +1,51 @@
+const DAPIClient = require('@dashevo/dapi-client');
+const RpcClient = require('@dashevo/dashd-rpc/promise');
+const getNetworkConfig = require('../../lib/test/getNetworkConfig');
+
+const networkConfig = getNetworkConfig();
+
+const fetch = require('node-fetch');
+
 describe('DAPI', () => {
-  // getBlockHash height:1
-  it('should respond data from chain');
+  networkConfig.inventory.masternodes.hosts.forEach((nodeName) => {
+    describe(`DAPI ${nodeName}`, () => {
+      const options = {
+        debug: false,
+        verbose: false,
+        errors: false,
+        warnings: false,
+        seeds: [{ ip: networkConfig.inventory._meta.hostvars[nodeName].public_ip }],
+      };
+      const dapi = new DAPIClient(options, 3000);
+      const config = {
+        protocol: 'http',
+        user: 'dashrpc',
+        pass: 'password',
+        host: networkConfig.inventory._meta.hostvars[nodeName].public_ip,
+        port: 20002,
+      };
+      const rpc = new RpcClient(config);
+      // getBlockHash height:1
+      it('should respond data from chain', async () => {
+        const blockHash = await dapi.getBlockHash(1);
+        const blockHashRpc = await rpc.getBlockHash(1);
+        expect(blockHash).to.be.an('string');
```

Comment:
What's the point of this check?

Reply:
only to be sure that blockHash is not undefined. will remove it